### DISCUSSION TITLE
Raspberry Pi (v1) model B

### DISCUSSION THREAD

**Discussion by jfangiolini** [dtcooper/raspotify#510] [General]
I can confirm raspotify works fine on the OG raspberry model B, both pre and post 0.31.4

**Comment by JasonLG1979** [comment]
I'll add it back with an asterisk like the Pi Zero but it might be a moot point if I can't get `librespot` to compile for ARMv6. The [`new-api`](https://github.com/librespot-org/librespot/tree/new-api) will not currently compile for ARMv6 and it is set to be merged into `dev` (what Raspotify is based on) if it turns out that it's just not possible to get it to compile for ARMv6, which is a possibility since ARMv6 is ancient, it may force my hand an I will have no choice but to drop ARMv6 support.
